backend/.env.example

```diff
@@ -1,16 +1,38 @@
-# Select your video service provider; the default is vonage.
-VIDEO_SERVICE_PROVIDER='vonage or opentok'
-# If you're using the Vonage Unified Platform, enter your credentials here.
-VONAGE_APP_ID=vonage-app-id
-VONAGE_PRIVATE_KEY=vonage-app-id
-# If you're using the legacy OpenTok Platform, enter your credentials here.
-OT_API_KEY=api-key
-OT_API_SECRET=api-secret
-# If you're deploying to Vonage Cloud Runtime, enter your port here; the default is 3345.
-VCR_PORT=vcr-port
-# If you're implementing the jira service for customer feedback, enter your credentials here.
-JIRA_URL=jira-url
-JIRA_API_URL=jira-api-url
-JIRA_TOKEN=jira-token
-JIRA_PROJECT_KEY=jira-project-key
-JIRA_COMPONENT_ID=jira-component-id
+# Get your Vonage Application ID and Private Key here: https://dashboard.nexmo.com/
+# If using OpenTok SDK and credentials make sure to remove this block and uncomment the block below with OT_ variables.
+
+# The default value for VIDEO_SERVICE_PROVIDER is 'opentok', make sure to change it to 'vonage' if using the Vonage environment.
```

Comment:
In line with my last comment, if we're planning to change this default value soon, I almost wouldn't mention the default here. In fact there's no real reason for this value to have a default, we can just make it mandatory and avoid further confusion